### DISCUSSION TITLE
ESD diode in SOD-523 package

### DISCUSSION THREAD

**Discussion by milador** [microsoft/jacdac#947] [DDK]
I am looking for an alternative ESD diode in larger package as those alternative options are all tricky to solder without pick and place.

Would the following diodes in SOD-523 package work?

- STS521050BL90 ( Current Peak is 1A instead of 4A )
- ESDBV5V0D5-TP ( Seems like a better option )

Thanks

**Comment by pelikhan** [comment]
@sehodges @mmoskal

**Reply by jamesadevine** [reply]
@milador With ESD diodes on the data line, we are particularly concerned about capacitance. Both of the presented options are an increase on our current part: RCLAMP0521P-N which has a capacitance of 0.4 pF. It looks like both of the options presented have a higher capacitance. 

Have you seen any diodes in your desired package with a particularly low capacitance?

**Reply by milador** [reply]
Hi @jamesadevine, how about the followings:

- ESD7371XV2T1G : 0.43pF @ 1MHz
- AQ3530-01LTG : 0.3pF @ 3GHz
- ESD7361XV2T5G : 0.55pF @ 1MHz 
- SZESD7361XV2T1G : 0.55pF @ 1MHz
- AQ3522-01LTG : 0.15 pF
- ESD5Z5VU : 0.4pF @ 1MHz

These have the lowest capacitance that I could find in SOD-523 package.

**Reply by sehodges** [reply]
capacitance-wise these are all fine

**Reply by milador** [reply]
@sehodges Thanks Steve, Would the higher Peak Pulse Current provide better protection as well? And any requirements for the Peak Pulse Power? I am not very familiar with ESD diodes and just looking at it as a zener diode. Apologies for all the question!

**Reply by sehodges** [reply]
Sorry @milador, only just got around to following up on this. I'm not super familiar with ESD diodes only and FWIW I look at them as a zener diodes too :-). As I said above, in terms of capacitance those diodes are all OK. The JD_DATA net will be OK with a few pF and these are all well below that. The JD_PWR net doesn't care about extra capacitance - in fact I'm not sure how much it needs ESD protection because it's already got a big cap on it and that'll supress ESD to some extent. 

I also checked the reverse breakdown voltages and they look OK too; I'd be tempted to choose one of the parts that has a lower value.

A larger peak pulse current and power is generally good because it means they can absorb more which is ultimately the job - absorb power when the voltage is too high. For an ESD diode we care about really high voltages (and very low currents) of course, so if anything I'd look at how many kV the diode is good for. The parts you suggest range from 8kV to 30kV and larger is obviously better. These parts can also degrade over time, so a higher rated part may withstand more lower voltage ESD events than a lower rated one. 

But at the end of the day you might also want to pick the cheapest part ;-).

**Reply by milador** [reply]
@sehodges We did few tests and the hardware looks fine as far as I can tell. Jacdac dashboard still doesn't detect the module. I am going over the bootloader code and the process of setting up the module id again in case I messed up something. The generated app works pretty well with jacdac prototype module and pressure sensor connected using jumper wires. Any suggestion or idea about what can be the issue? Thanks

**Comment by mmoskal** [comment]
You can also skip them altogether, especially for prototyping. I have not seen any issues with that, but then again I'm a software not hardware engineer.

**Reply by milador** [reply]
@mmoskal That would be another option, but hopefully I can find an alternative diode as I have the design based on that package.

**Reply by milador** [reply]
Actually another design with the diode mentioned in the documents as well.

**Reply by sehodges** [reply]
yeah for prototyping it's fine not to have ESD diodes. but the reason why they exist and why products use them is because if you don't fit them then too many customers end up with a bad experience and it ends up costing more thru replacements and bad reputation. at the end of the day it's all market economics - this isn't just some nice-to-have that makes the engineer feel good :-).

of course it's possible that some of our designs will be robust to ESD for other reasons - like when we use Padauk MCUs based on older silicon processes with bigger geometries that might not be so susceptible to ESD, or when we make a micro:bit Jacdapter and we know that micro:bit has been ESD tested already. it's also possible that modules are so cheap that customers will just throw them away and replace should they have a failure. but without some testing and/or in-market experience to back up these possibilities, I am currently thinking it's best for us to follow the common industry practice and put ESD protection on Jacdac ports

**Reply by milador** [reply]
@sehodges Thanks for providing more information. I agree and I think it's the best to include them due to reasons you mentioned.  The cost of adding them in large quantities is very little so it definitely doesn't hurt to add them. My final design uses RCLAMP0521P-N diode which should be fine using pick and place assembly. I have the second design ( maker friendly ) which uses parts with larger footprint which is optimized for smd oven , but I still would like to have it as close as possible to the final version. I think I will stick with ESD diodes in SOD-523 package for this round and not populate the diodes if I run to issues.
